Assets/Scripts/Drinkables/AbstractDrinkable.cs

```diff
@@ -20,8 +20,7 @@ protected void CommonDrunkennessEffects()
         {
             // All beers will have a base score multiplier of 2
             Indestructibles.PlayerData.ScoreMultiplier *= 2;
-            if (Indestructibles.PlayerData.IntoxicationLevel < 1.0f)
```

Comment:
why did you remove the if? the camera settings can't go above 1 and this is the workaround to prevent the camera from being completely sober when the player still has beers in their system